podcasts/MainTabBarController.swift

```diff
@@ -585,6 +585,19 @@ class MainTabBarController: UITabBarController, NavigationProtocol {
             controller.present(whatsNewViewController, animated: true)
         }
     }
+
+    // There are different areas of the app that relies on presenting VCs from the tab bar
+    // However, sometimes the tab bar is already displaying the player.
+    // This code simple checks if the tab bar is already presenting something and, if yes,
+    // present the VC through the presentedViewController
+    override func present(_ viewControllerToPresent: UIViewController, animated flag: Bool, completion: (() -> Void)? = nil) {
+        if FeatureFlag.newPlayerTransition.enabled, let presentedViewController {
```

Comment:
Note: I made some comments below about an alternative to presenting on top of the presented view.

If I enable 'Open Player Automatically' in the General settings and then:

1. Open a Podcast
2. Tap on an episode to see its details
3. Tap Play
4. I see: `Attempt to present <podcasts.PlayerContainerViewController: 0x11982c800> on <podcasts.EpisodeDetailViewController: 0x11782f600> (from <podcasts.EpisodeDetailViewController: 0x11782f600>) whose view is not in the window hierarchy.`

I did some digging and the isBeingDismissed flag is set to true and checking it fixes this for me.

```suggestion
        if FeatureFlag.newPlayerTransition.enabled, let presentedViewController, !presentedViewController.isBeingDismissed {
```

Reply:
I am still able to reproduce this in the current build, but applying the isBeingDismissed check fixes it.